ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGeneratorSpectraS3InitRequests_Test.java

```diff
@@ -258,4 +258,66 @@ public void testGenerateInitSpectraS3RequiredBoolQueryParamOmittedFromFunctionSi
                 + "}";
         assertEquals(expectedOutput, output);
     }
+
+    @Test
+    public void testGenerateInitSpectraS3CreatePutJobRequest_WithRequiredOperationQueryParam_AndPriorityEnumOptionalQueryParam()
+            throws IOException, TemplateModelException {
+        final Map<String,Object> testMap = new HashMap<>();
+        final Request testRequest = RequestConverter.toRequest(new Ds3Request(
+                "com.spectralogic.s3.server.handler.reqhandler.spectrads3.job.CreatePutJobRequestHandler",
```

Comment:
There is a test utility that creates some of the special cased requests, including CreatePutJobRequestHandler so you don't have to create the data structure here. Its in ds3-autogen-test-util package:
com.spectralogic.ds3autogen.testutil.Ds3ModelFixtures.getRequestBulkPut()